export_ova.sh

```diff
@@ -21,4 +21,4 @@ if [[ -e $EXPORTVMNAME.ova ]]; then
 fi
 
 $VBOX export $VMNAME --output $EXPORTVMNAME.ova
-md5sum $EXPORTVMNAME.ova > $EXPORTVMNAME.ova.md5sum
+shasum -a 1 $EXPORTVMNAME.ova > $EXPORTVMNAME.ova.sha1
```

Comment:
Actually misread that.... I think sha1sum is installed by default, shasum maybe not?